src/topology/tactic.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2020 Reid Barton. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Reid Barton
+-/
+import tactic.auto_cases
+import tactic.tidy
+import tactic.with_local_reducibility
+import topology.basic
+
+/-!
+# `continuity` tactic
```

Comment:
Since the file is named `topology/tactic.lean`, it probably makes sense to have a more generic module doc.
```suggestion
/-!
# Tactics for topology

Currently we have one domain-specific tactic for topology: `continuity`.

-/

/-!
### `continuity` tactic
```